### DISCUSSION TITLE
Inquiry: Feasibility of Restoring and Colorizing Old Photos with Fooocus AI

### DISCUSSION THREAD

**Discussion by crazyanageek** [lllyasviel/Fooocus#1565] [Q&A]
Hello,

I am exploring the potential of using AI for restoring and colorizing old photographs and came across your project. I'm particularly interested in understanding if an AI image-generating tool like Fooocus can be employed for this purpose.

My questions are as follows:

1. Is it technically feasible to restore and colorize old photographs using Fooocus or similar AI image-generating tools? If so, what are the general steps or processes involved?

2. Are there specific algorithms or models (Lota)  within Fooocus that are best suited for photo restoration and colorization tasks?

Understanding these aspects will greatly aid in my project, and any insights or guidance you can provide would be highly valuable. Thank you for your time and assistance.

Best regards,

CAG

**Comment by mashb1t** [comment]
Hey @crazyanageek,
recolor adapters have not (yet) been implemented in Fooocus (official). In the meantime you can check out the recolor feature in the fork https://github.com/fenneishi/Fooocus-ControlNet-SDXL or find controlnet models here to use them in ComfyUI / Automatic1111, where recoloring is already supported:
https://huggingface.co/lllyasviel/sd_control_collection/tree/main => sai_xl_recolor_XXXlora.safetensors
